script/Ciel/Day 3/03_01_CIEL03_3_3C.txt

```diff
@@ -493,10 +547,10 @@ I don't know what tomorrow holds.
 [sha:29c94f5a15d86a99b174bd9f62aef8226796fd11]{
 -- Page 279, Offset 5567.
 -- 分からないけど、もし許されるのなら。
- I don't know, but if I can―――
+I don't know, but if I can hope for one thing―――
 }
 [sha:1813b78dbb8bea977d1dba337b76807622017126]{
 -- Page 279, Offset 5568.
 -- せめて明日、今までお世話になった人たちにお礼を言うだけの時間が、あってくれればいいと願った。
-I hope that tommorow I will have enough time to thank the people who have helped me.
+I hope that I at least have enough time to thank all of the people that have helped me over the years.
```

Comment:
The end. what a nice young man! I'm glad the story ended right here and nobody else got hurt